ui/components/multichain/account-overview/account-overview-tabs.tsx

```diff
@@ -112,7 +112,7 @@ export const AccountOverviewTabs = ({
             data-testid="account-overview__asset-tab"
             {...tabProps}
           >
-            <Box marginTop={2} marginBottom={2}>
+            <Box marginBottom={2}>
```

Comment:
Modified padding and margin on this page so that the activity bar is aligned across pages without any extra hacks.